### ISSUE TITLE
No completion on abstract field forwarding

### ISSUE THREAD

**Issue by jaypavlina** [TiVo/intellij-haxe#30]
See a detailed description of the feature in [the Haxe Manual.](http://haxe.org/manual/types-abstract-forward.html)

```
@:forward(length) // unresolved symbol
abstract Hello(String) {
    public function new() {
        this = "hello";
    }
}
var hello = new Hello();
hello.length; // unresolved symbol, no completion
```

In the above example, only the field length should be forwarded and should have completion. You can also write @:forward with no arguments to forward all fields.

TIR: STB-7291

**Comment by bynuff** [comment]
@jaypavlina this issue was fixed in #448 PR. Please update haxe plugin to latest version.

**Comment by bynuff** [comment]
@EBatTiVo can you close this issue?
